docs/01-start-here/06-testing-tips.md

```diff
@@ -42,6 +49,8 @@ To run only the tests for a particular test suite, use the command:
   test-only *<name-of-your-test-suite>
 ```
 
+## Testing AMP
```

Comment:
@ScottPainterGNM makes sense to do a separate PR. Certainly I think it would be good to combine them, but I will trust both of your judgments 😄